base/record_replay.cc

```diff
@@ -243,6 +243,10 @@ bool HadMismatch() {
   return V8RecordReplayHadMismatch();
 }
 
+bool HasAsserts() {
+  return !AreAssertsDisabled() && IsRecordingOrReplaying();
```

Comment:
really seems like this should be calling `V8HasAsserts` so we keep the implementation in a single place.